.github/workflows/update-python.yml

```
@@ -0,0 +1,31 @@
+---
+on:  # yamllint disable-line rule:truthy
+  pull_request_target:
+    types: [opened, reopened]
+    paths:
+      - .python-version
+
+permissions:
+  pull-requests: write
+
+jobs:
+  update-python:
+    runs-on: ubuntu-latest
+    name: Update Python version used by renovate
+    steps:
+      - name: Post a reminder to update the Python version used by renovate
+        shell: python
+        # yamllint disable rule:line-length
+        run: |
+          import requests
+
+          PR_NUM = ${{ github.event.pull_request.number }}
+          REPO_SLUG = ${{ github.repository }}
```

@nre-ableton I guess it should be
```suggestion
          REPO_SLUG = "${{ github.repository }}"
```
to make `REPO_SLUG` a proper string.

---

Yes, we need to do this in all Ableton Ansible OSS roles. Kinda sucks that we didn't notice this before copy/pasting the file everywhere. :sweat_smile: I made an internal issue for us to de-duplicate this by using a shared workflow.